### PR TITLE
Add some instructions about running integration tests

### PR DESCRIPTION
**Pull request by strk**
Derived from
https://matrix.org/_matrix/media/v1/download/matrix.org/KPnyFigwPBbKYZArIYKeCMHj
by @sapk

### CODE REVIEW

**Comment by strk** [comment]
@sapk please review

File: integrations/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  make test-sqlite
+
+# Running individual tests
+
+Example command to run GPG test with sqlite backend:
+
+```
+go test -c code.gitea.io/gitea/integrations \
+  -o integrations.sqlite.test -tags 'sqlite' &&
+  GITEA_ROOT="$GOPATH/src/code.gitea.io/gitea"
```
Comment (lafriks) [inline integrations/README.md]: Shouldn't this line also end with `\`?

**Comment by strk** [comment]
> Shouldn't this line also end with `\`?

Good catch! Fixed with 3ea106e3

**Review by sapk — APPROVED** [review]

**Comment by lunny** [comment]
but the command cannot run in `integrations` directory?

**Comment by sapk** [comment]
I would suggest to do a `make clean && make build` before to be sure that a previous failed test doesn't interfer and the `make build` is because some test (for example ssh server) use the binary ./gitea. Maybe we should fix that to use the test binary in place in the future.

**Comment by strk** [comment]
a1330c02 suggests the clean build

**Review by techknowlogick — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Comment by sapk** [comment]
For information, CI fail is unrelated (git failed to clone).

**Pull request merged**